lambda/src/redirect.js

```diff
@@ -0,0 +1,32 @@
+const streamClient = require('./streamClient');
+
+const sendClickEvent = (outboundUrl) => {
+  return streamClient.publish('link-clicked', {
+    outboundUrl: outboundUrl
+  }).catch(() => Promise.reject({status: 500, message: 'Failed to publish event'}));
+};
+
+const redirect = (queryStringParameters) => {
+  if (queryStringParameters && queryStringParameters['outboundUrl']) {
+    sendClickEvent(queryStringParameters['outboundUrl'])
+      .catch((e) => console.log(e));
+    return {
+      "statusCode": 302,
+      "headers": {
+        "Location": queryStringParameters['outboundUrl']
+      },
+      "body": null,
+      "isBase64Encoded": false
+    }
+  }
+  return {
```

Comment:
I know it's a client error, but it might be worth logging something here, just to highlight the fact that something somewhere is generating invalid redirector URLs. E.g. `console.error('Redirector received invalid URL blah blah')`